wasm.md

```diff
@@ -817,19 +849,21 @@ Currently, in the expanded form, the `export`s will come after the definition of
  // -------------------------------------------------------------------
     rule  asLocalType(LDECLS) => #asLocalType(LDECLS, .ValTypes)
 
-    rule #asLocalType(.LocalDecls          , VTYPES) => [ VTYPES ]
-    rule #asLocalType(local VTYPES' LDECLS , VTYPES) => #asLocalType(LDECLS , VTYPES + VTYPES')
+    rule #asLocalType(.LocalDecls                                            , VTYPES) => [ VTYPES ]
+    rule #asLocalType(local               VTYPES':ValTypes LDECLS:LocalDecls , VTYPES) => #asLocalType(LDECLS , VTYPES + VTYPES')
+    rule #asLocalType(local ID:Identifier VTYPE:AValType   LDECLS:LocalDecls , VTYPES) => #asLocalType(LDECLS , VTYPES + { ID VTYPE } .ValTypes)
 ```
 
 ### Function Implicit Type Declaration
 
 It could also be declared implicitly when a `TypeUse` is a `TypeDecls`, in this case it will allocate a type when the type is not in the current module instance.
+One undefined behaviour should we keep the identifiers from unfolded declarations or not?
```

Comment:
Not sure what you mean here. Either way, it looks like it should be a **TODO**, not part of the documentation.